be/test/exec/tablet_sink_test.cpp

```diff
@@ -935,6 +935,11 @@ TEST_F(OlapTableSinkTest, decimal) {
 }
 
 int main(int argc, char* argv[]) {
+    std::string conffile = std::string(getenv("DORIS_HOME")) + "/conf/be.conf";
+    if (!doris::config::init(conffile.c_str(), false)) {
+        fprintf(stderr, "error read config file. \n");
+        return -1;
+    }
```

Comment:
you can set config value in SetUp(), otherwise you can only run this test in run-ut.sh